inference_rules.adoc

```diff
@@ -135,24 +135,15 @@ described in the table below.
 
 |===
 |Scenario |Query Generation |Duration |Samples/query |Latency Constraint |Tail Latency | Performance Metric
-|Single stream |LoadGen sends next query as soon as SUT completes the previous query | 1024 queries and 600 seconds |1 |None |90% | 90%-ile measured latency
-|Server |LoadGen sends new queries to the SUT according to a Poisson distribution |270,336 queries and 600 seconds |1 |Benchmark specific |99% | Maximum Poisson throughput parameter supported
+|Single stream |LoadGen sends next query as soon as SUT completes the previous query | 600 seconds |1 |None |90%* | 90%-ile early-stopping latency estimate
+|Server |LoadGen sends new queries to the SUT according to a Poisson distribution | 600 seconds |1 |Benchmark specific |99%* | Maximum Poisson throughput parameter supported
 |Offline |LoadGen sends all samples to the SUT at start in a single query | 1 query and 600 seconds | At least 24,576 |None |N/A | Measured throughput
-|Multistream | Loadgen sends next query, as soon as SUT completes the previous query | 270,336 queries and 600 seconds | 8 | None | 99% | 99%-ile measured latency|
+|Multistream | Loadgen sends next query, as soon as SUT completes the previous query | 600 seconds | 8 | None | 99%* | 99%-ile early-stopping latency estimate|
 |===
 
-The number of queries is selected to ensure sufficient statistical confidence in
-the reported metric. Specifically, the top line in the following table. Lower
-lines are being evaluated for future versions of MLPerf Inference (e.g., 95%
-tail latency for v0.6 and 99% tail latency for v0.7).
+An early stopping critereon (described in more detail in <<appendix-early_stopping>>) allows for runs with relatively few number of processed queries to be valid, with the penalty that the effective computed percentile will be slightly higher.  This penalty counteracts the increased variance inherent to runs with few queries, where there is a higher probability that a particular run will, by chance, report a lower latency than the system should reliably support.
```

Comment:
Minor copy edits: 

"criterion" rather than "criterion" 

"a relatively small number of processed queries" or "relatively few processed queries"

Reply:
Fixed in most recent commit.